docs/tutorial/node/src/from_one_to_multiple_tiles.md

```diff
@@ -0,0 +1,33 @@
+# From One To Multiple Tiles
+
+After modeling a single tile, let's create a grid of them. For the grid to be our game board, we need two features:
+
+1. A data model: This shall be an array where each element describes the tile data structure, such as the
+   url of the image, whether the image shall be visible and if this tile has been solved. We modify the model
+   from JS code.
+2. A way of creating many instances of the tiles, with the above `.slint` markup code.
+
+In Slint we can declare an array of structures using brackets, to create a model. We can use the <span class="hljs-keyword">for</span> loop
+to create many instances of the same element. In `.slint` the for loop is declarative and automatically updates when
+the model changes. We instantiate all the different <span class="hljs-title">MemoryTile</span> elements and place them on a grid based on their
+index with a little bit of spacing between the tiles.
+
+First, we copy the tile data structure definition and paste it at top inside the `memory.slint` file:
+
+```slint
+{{#include ../../rust/src/main_multiple_tiles.rs:tile_data}}
```

Comment:
do you mean to include a rust file here?

Reply:
@ogoffart I see now we have the same issue on the C++ Version.

Reply:
But it's references Slint code inside of that file. Therefore it's ok output looks also good.